smoke-tests/apps/JmxMetric/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/JmxMetricTest.java

```diff
@@ -140,7 +136,7 @@ private void verifyJmxMetricsSentToOtlpEndpoint() {
 
   private void verifyJmxMetricsSentToBreeze() throws Exception {
     List<Envelope> metricItems =
-        testing.mockedIngestion.waitForItems(JmxMetricTest::isJmxMetric, 1, 10, TimeUnit.SECONDS);
+        testing.mockedIngestion.waitForItems(JmxMetricTest::isJmxMetric, 7, 10, TimeUnit.SECONDS);
```

Comment:
(although that would be 6?)
```suggestion
        testing.mockedIngestion.waitForItems(JmxMetricTest::isJmxMetric, jmxMetricsAllJavaVersionsBreeze.size(), 10, TimeUnit.SECONDS);
```